### ISSUE TITLE
lower field text is not centered

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#461]
```
The help text is not well centered, its position varies if there are buttons on 
either sides.
```

Original issue reported on code.google.com by `vittorio...@gmail.com` on 1 Nov 2012 at 9:01

**Comment by GoogleCodeExporter** [comment]
Original comment by `sheepyluva` on 15 Jan 2013 at 5:33
* Added labels: **Component-QtFrontend**

**Comment by GoogleCodeExporter** [comment]
```
this got fixed in r2c233349df21 but got broken again in r869f80966a77
```

Original comment by `vittorio...@gmail.com` on 18 Jan 2013 at 11:53

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 286823b10f35.
```

Original comment by `drew1313...@gmail.com` on 20 Jan 2013 at 1:01
* Changed state: **Fixed**
